grpc/pom.xml

```diff
@@ -54,22 +54,22 @@
         <dependency>
             <groupId>io.grpc</groupId>
             <artifactId>grpc-netty-shaded</artifactId>
-            <version>1.31.1</version>
+            <version>1.49.0</version>
```

Comment:
Seems like all these bumps were necessary due to the protoc/protoc-gen-grpc-java changes you needed to do? If so, these are unavoidable + I'm fine merging, but a bit of a sanity check would be nice to be sure that a GRPC version bump doesn't affect response times (esp with all of the custom GRPC knob turning we've done 😅 )

I think it would be good enough to just run the functional test github action off of this branch, and sanity check the logs of the `Run functional test queries` + compare its runtime to the same CI step on `original-direction`. If things look ~ the same, we should be good + can just monitor larger runs that use this updated GRPC lib in the future

Reply:
Ran the functional test on this branch, results at https://github.com/replicahq/graphhopper/actions/runs/3039003177/jobs/4893417723. Runtime seems ok - 7m 28s. On the last commit to `original-direction`, this step took 7m 16s; on the commit before that, it was 7m 22s. Logs look similar across the three runs.

FYI, the [Analyze functional test results by comparing to golden OD response set](https://github.com/replicahq/graphhopper/actions/runs/3024800705/jobs/4866686222#logs) step is currently failing on `original-direction` and also fails on my branch. Looks like it started failing with commit https://github.com/replicahq/graphhopper/commit/550be5ae6fc2286a03a123e96b61f19b71021153 which changed the profile used for cars. Maybe this is expected and our golden set needs updating?

Reply:
Seems close enough to me! 🚀 Yeah, the comparison to "golden set" results failing in the functional test is almost definitely expected with all the profile changes we're getting through recently - I think once this current batch of changes settles down, we can regenerate golden set results for ✅  in the future